docs/src/experiment_setup_guide.md

```diff
@@ -0,0 +1,220 @@
+This is the ClimaAtmos configuration which will be used to run all simulations in the calibration. The only changes between model runs are the parameters selected for calibration.
+This file is a typical Atmos YAML file with a few restrictions. It must:
+
+- be named `atmos_config.yml`. 
+- include an `output_dir` to store model output.
+- include the path to the restart file. This is detailed in the next section.
+Here is an example of the arguments you will need:
+```
+output_dir: output/experiment_name
+restart_file: experiments/experiment_name/restart_file.hdf5 
+```
+
+If your configuration requires a parameter TOML file, add this file to the experiment folder and set the `toml` entry in the configuration to the relative file path from the base directory of `CalibrateAtmos.jl`.
+
+!!! note
+    When targeting a global or otherwise costly simulation, it may be worth it to optimize your timestep `dt` as well as your timestepping algorithm, `ode_algo`.
+
+## Restart File
+The restart file is a model snapshot which will be used to spin-off all ensemble members' forward model runs.
+
+Once you have settled on a configuration, follow these steps to generate a restart file:
+
+1. Run ClimaAtmos with your configuration to determine the time it takes for the simulation to reach an equilibrium state. 
+2. Generate a restart file by setting `dt_save_restart` in the model configuration to the equilibrium timestamp. 
+3. Transfer the file to your experiment directory and enter the relative path into your atmosphere configuration file:
+```
+restart_file: experiments/experiment_name/restart_file.hdf5 
+```
+
+!!! note
+    A restarted simulation starts from the saved date of the restart file. A restart file saved at 200 days and a `t_end` of 201 days will only run for 1 simulated day:
+    ```
+    restart_file: experiments/experiment_name/day200.hdf5 
+    t_end: 201 days
+    ``` 
+
+## Prior Distribution File
+First, create your TOML file in your experiment folder.
+For each calibrated parameter, create a prior distribution with the following format:
+```toml
+[long_name]
+alias = "alias_name"
+type = "float"
+prior = "Parameterized(Normal(0,1))"
+constraint = "[bounded(0,5)]"
+```
+Note that the prior distribution here is in unconstrained space - the `constaint` list constrains the distribution in parameter space.
+
+An easy way to generate prior distributions in constrained parameter space is with the [constrained_gaussian](https://clima.github.io/EnsembleKalmanProcesses.jl/dev/API/ParameterDistributions/#EnsembleKalmanProcesses.ParameterDistributions.constrained_gaussian) constructor from `EnsembleKalmanProcesses.ParameterDistributions`. Here is an example:
+```julia
+using EnsembleKalmanProcesses.ParameterDistributions
+mean = 125
+standard_deviation = 40
+lower_bound = -50
+upper_bound = Inf
+constrained_gaussian("name", mean, standard_deviation, lower_bound, upper_bound)
+```
+This will output a ParameterDistribution in unconstrained space:
```

Comment:
The constructor will provide a Normal distribution, which, when transformed will approximate the target mean and standard deviation provided